dropshot_endpoint/src/lib.rs

```diff
@@ -1104,4 +1152,103 @@ mod tests {
             Some("Endpoint requires arguments".to_string())
         );
     }
+
+    #[test]
```

Comment:
Nice tests.

Reply:
Note the use of my dope schema crate! ;-)